erratatool.yml

```diff
@@ -9,7 +9,7 @@ brew_tag_product_version_mapping:
 cdn_repos:
 - rhel-7-server-ose-{MAJOR}_DOT_{MINOR}-rpms__x86_64
 - rhel-7-for-power-le-ose-{MAJOR}_DOT_{MINOR}-rpms__ppc64le
-- rhel-7-for-power-9-ose-{MAJOR}_DOT_{MINOR}-rpms__ppc64le 
+- rhel-7-for-power-9-ose-{MAJOR}_DOT_{MINOR}-rpms__ppc64le
```

Comment:
Actually, both power-le and power-9 can be removed; 3.11 is x86-only now.

Reply:
Removed.